yt_dlp/extractor/tubitv.py

```diff
@@ -108,3 +113,30 @@ def _real_extract(self, url):
             'uploader_id': video_data.get('publisher_id'),
             'release_year': int_or_none(video_data.get('year')),
         }
+
+
+class TubiTvShowIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?tubitv\.com/series/[0-9]+/(?P<show_name>\S*)(?:\?\S*)'
+    _TESTS = [{
+        'url': 'https://tubitv.com/series/3936/the-joy-of-painting-with-bob-ross?start=true',
+        'playlist_mincount': 390,
+        'info_dict': {
+            'id': 'the-joy-of-painting-with-bob-ross',
+        }
+    }
+    ]
+
+    def _entries(self, show_url, show_name):
+        show_webpage = self._download_webpage(show_url, show_name).replace('undefined', '\"undefined\"')
+        show_json = self._parse_json(self._search_regex(
+            r"window\.__data\s*=\s*({.+?});\s*</script>",
+            show_webpage, 'data'), show_name)['video']
```

Comment:
Is it a `js` or `json`? If `js`, use `self._parse_json(... , transform=js_to_json)`

```py
show_webpage = self._download_webpage(show_url, show_name).replace('undefined', '\"undefined\"')
```
should probably be implemented in `js_to_json` too